tutorials/Example_Track_on_Radar_Segment_on_Satellite/track_on_radar_segment_on_satellite.ipynb

```diff
@@ -0,0 +1,1245 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "## You are using the Python ARM Radar Toolkit (Py-ART), an open source\n",
+      "## library for working with weather radar data. Py-ART is partly\n",
+      "## supported by the U.S. Department of Energy as part of the Atmospheric\n",
+      "## Radiation Measurement (ARM) Climate Research Facility, an Office of\n",
+      "## Science user facility.\n",
+      "##\n",
+      "## If you use this software to prepare a publication, please cite:\n",
+      "##\n",
+      "##     JJ Helmus and SM Collis, JORS 2016, doi: 10.5334/jors.119\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pyart\n",
+    "import xarray as xr\n",
+    "import s3fs\n",
+    "import numpy as np\n",
+    "import datetime\n",
+    "import matplotlib.pyplot as plt\n",
+    "import cartopy.crs as ccrs\n",
+    "from matplotlib.pyplot import cm as cmaps\n",
+    "import pandas as pd"
```

Comment:
We should highlight any packages that are not in the tobac requirements (e.g. pyart) so that the user knows to install them